server/index.js

```diff
@@ -2,23 +2,23 @@
 const express = require('express');
 // const path = require('path');
 const cors = require('cors');
+const packFeed = require('./controllers/packFeed.js');
```

Comment:
the way that Louise set up rotes, this should be imported into  and exported out of controllers/index, then in server/routes, you will set up the req handler for the pack feed